src/PrestaShopBundle/Resources/config/routing/admin/sell/orders/orders.yml

```diff
@@ -0,0 +1,45 @@
+admin_orders_index:
+  path: /
+  methods: [GET]
+  defaults:
+    _controller: PrestaShopBundle:Admin/Sell/Order/Order:index
+    _legacy_controller: AdminOrders
+
+admin_orders_search:
+  path: /
+  methods: [POST]
+  defaults:
+    _controller: PrestaShopBundle:Admin/Sell/Order/Order:search
+    _legacy_controller: AdminOrders
```

Comment:
Let's not forget _legacy_links

Reply:
Yeah, but after full page migration only. :wink: